game/plugins/src/main/kotlin/gg/rsmod/plugins/content/combat/specialattack/weapons/saradominsword/saradomin_sword.plugin.kts

```diff
@@ -0,0 +1,58 @@
+package gg.rsmod.plugins.content.combat.specialattack.weapons.saradominsword
+
+import gg.rsmod.plugins.content.combat.dealHit
+import gg.rsmod.plugins.content.combat.formula.MeleeCombatFormula
+import gg.rsmod.plugins.content.combat.specialattack.SpecialAttacks
+
+val SPECIAL_REQUIREMENT = 100
+val SARASWORD_SPEC_ANIMATION = 11993
+val SARASWORD_SPEC_PLAYER_GFX = 2115
+val SARASWORD_SPEC_TARGET_GFX = 1194
+val MAGIC_DAMAGE_MAX_HIT = 20.0
+val MAGIC_DAMAGE_MIN_HIT = 5.0
+val SARASWORD_SPEC_SFX_ID1 = 3887 // According to https://www.runelister.com/forum/topic/osrs-sound-effect-list/
```

Comment:
Saradomin sword was not in OSRS on release and was later added with GWD. As a result the IDs for any content that was added to osrs after launch most likely wouldnt match up.

I did a quick search, it appears that rather than using id 3887, the correct sound should be 3853. Im not sure if there was the SFX2 used by SS back then.

Reply:
Yes 3853 is the correct sound id. The ambient sound is included in sound 3853.

Reply:
Perfect, thanks guys. I couldn't track that down before. Couldn't find any videos to compare the SFX IDs to.